dart/neural/BackpropSnapshot.cpp

```diff
@@ -2324,7 +2342,7 @@ Eigen::MatrixXs BackpropSnapshot::finiteDifferenceVelVelJacobian(
 
   Eigen::VectorXs originalVel = world->getVelocities();
 
-  s_t EPSILON = 1e-7;
+  s_t EPSILON = 1e-9;
```

Comment:
This makes me nervous. In general, my experience is that 1e-9 is too small and starts to run into numerical errors. Do the tests not pass at 1e-7?

Reply:
No, I think this is a very ancient modification since I want to test whether the epsilon will affect the result.